src/utils/sentry.test.js

```diff
@@ -0,0 +1,53 @@
+import { getAppDetails } from './sentry';
+
+describe('getAppDetails function', () => {
+  beforeEach(() => {
+    Object.defineProperty(global, 'location', {
```

Comment:
@adonispuente Jest does not reset global variables automatically after a test has executed, so with this change we run the risk of contaminating tests that run after this spec.

There are a few ways to avoid global variables in the spec. In lieu of accessing `window.location` in the source code for `sentry.ts` you can add a helper function that simply returns the value of window.location, and then spy on that function in your test.

An alternative would be to clean up after this test in an afterEach.

Reply:
I sent up the change in the latest commit!